tests/admin/load-tests.php

```diff
@@ -183,7 +186,7 @@ public function test_perflab_get_modules() {
 	public function test_perflab_get_module_data() {
 		// Use test data directory with demo modules that match the modules declared on top of this file.
 		foreach ( self::$demo_modules as $module_slug => $expected_module_data ) {
-			$module_data = perflab_get_module_data( TESTS_PLUGIN_DIR . '/tests/testdata/demo-modules/' . $module_slug . '/load.php' );
+			$module_data = perflab_get_module_data( TESTS_PLUGIN_DIR . '/tests/testdata/demo-modules/' . $module_slug . '/load.php', $module_slug );
```

Comment:
See my above comment, I think it would be better to keep the parameters for that function as is and handle getting the focus and module slug internally.

Reply:
👍🏼  done